message_parser/src/test/java/cumulus_message_adapter/message_parser/MessageParserTest.java

```diff
@@ -104,6 +104,49 @@ public void testLoadAndUpdateRemoteEvent() {
         }
     }
 
+    @Test
+    public void testLoadAndUpdateRemoteEvent_s6Success() {
```

Comment:
I actually removed this test case because it didn't end up testing anything for us, was useful in narrowing down the actual issue. We can add it back in if you'd like the coverage.